can/interface.py

```diff
@@ -21,6 +24,27 @@
     'neovi':            ('can.interfaces.neovi_api', 'NeoVIBus')
 }
 
+hookimpl = HookimplMarker('pythoncan')
+
+
+@hookimpl(trylast=True)
+def pythoncan_interface(interface):
+    """This hook is used to process the initial config
+    and possibly input arguments.
+    """
+    return BACKENDS.get(interface)
+
+
+def get_pluginmanager(load_entrypoints=True):
+    pm = PluginManager("pythoncan")
+    pm.add_hookspecs(hookspecs)
+    # XXX load internal plugins here
+    pm.register(sys.modules[__name__])
```

Comment:
Can you help me understand this code & comment? Is this _just in case_ we wanted to define other hookspecs in the current namespace and have them self register into the plugin manager? Or does this pick up implementations/plugins?